### ISSUE TITLE
[ULDUAR] Thorim - Timers likely overlapped

### ISSUE THREAD

**Issue by therealCrysis** [Stormforge-gg/Bugtracker-Fordring#1879]
Spell timers seem to be overlapped as well
Sif's cast Frost Nova & Frost Bolt Volley almost instantly after each other. It gives 2x 15k damage at the same time which is almost 100% wipe.

Frost Bolt Volley should be 8s cooldown (not sure about Frost Nova, yet these spells should not be casted simultaneously)

**Comment by Aithne99** [comment]
Frostbolt Volley is on a 10 second timer, and yes, its cooldown is theoretically 8 seconds, but that cooldown seems like it should not affect Blink/Frost Nova. So please describe the EXACT problem you had with the encounter, in plain English.

**Comment by Aithne99** [comment]
And what the hell about Thorim?

**Comment by Godmatik** [comment]
Dunno about the spell cooldowns - but just reviewing old VODs -  Frost Nova and Frostbolt Volley should not instantly cast over each other, 

Looks like the general pattern is: Blink --> Frostbolt --> Frost Nova --> 2 or 3 Frostbolts --> Frostbolt Volley --> 2 or 3 Frostbolts --> Blink --> repeat (EDIT: She also casts Blizzard sometime in that cycle)

https://www.youtube.com/watch?v=NKejKuxyE1M&ab_channel=JoeM

**Comment by Aithne99** [comment]
Altered it somewhat, it's live on the PTR. It is currently not very easy to determine when she should cast blink -> frost nova (that part of her spellcasting is fixed, she'll always immediately frost nova after blinking and not any other time), perhaps it is determined based on player positioning, but idk.

**Comment by Aithne99** [comment]
Closing so that it shows up in the bugtracker feed and people can check it out.

**Comment by adambcurrier** [comment]
Sif is dealing 25 man damage values in the 10 man version of the raid.
